sessionctx/variable/tidb_vars.go

```diff
@@ -112,6 +112,9 @@ const (
 	// tidb_record_plan_in_slow_log is used to log the plan of the slow query.
 	TiDBRecordPlanInSlowLog = "tidb_record_plan_in_slow_log"
 
+	// tidb_enable_slow_log enables TiDB to log slow queries.
+	TiDBEnableSlowLog = "tidb_enable_slow_log"
```

Comment:
I still have some concerns about adding a variable. When we think that the variable is inappropriate, we cannot easily remove it, which will cause compatibility problems. It doesn't matter if you just add an item to the configuration file. It's better that we only add a configuration item. This also meets the requirements of PM, and there is no risk. :)

Reply:
Mysql has similar variable too.
```mysql
mysql> show variables like '%slow_query_log%';
+---------------------+-----------------------------------------------------------+
| Variable_name       | Value                                                     |
+---------------------+-----------------------------------------------------------+
| slow_query_log      | OFF                                                       |
| slow_query_log_file | /usr/local/var/mysql@8.0/fengliyuans-MacBook-Pro-slow.log |
+---------------------+-----------------------------------------------------------+
2 rows in set (0.00 sec)
```

Reply:
Should we use the same name as MySQL?

Reply:
I'd like to.